### PR TITLE
Best Decision tree and Best MLP

### PR DESCRIPTION
**Pull request by Jacobian8**
Created the files for the 2 best models we needed. 
I also generated trees for the both the base and best

### CODE REVIEW

File: a1/a1/bestmlp.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from sklearn.neural_network import MLPClassifier
+from sklearn.model_selection import GridSearchCV
+from a1 import util
+
+# Grid search hyperparameters
+# I'm guessing it has to do with the combination of all 3 parameters but sometimes 
+# we get "ConvergenceWarning: <ACTIVATION_TYPE>: Maximum iterations (200) reached and the optimization hasn't converged yet."
+param_grid = {
+    'activation' : ['logistic', 'tanh', 'relu', 'identity'], # Requirement (Note: logistic == sigmoid)
```
Comment (FGRCL) [inline a1/a1/bestmlp.py:9]: I'm guessgin logistic is the same as sigmoid?

**Review by FGRCL — APPROVED** [review]

File: a1/a1/bestmlp.py (hunk 1 above)
Comment (Jacobian8) [reply a1/a1/bestmlp.py:9]: yessirrr apprently so
